### ISSUE TITLE
Add method IntervalBox(X, n)

### ISSUE THREAD

**Issue by dpsanders** [JuliaIntervals/IntervalArithmetic.jl#172]
For an IntervalBox X, should give X^n

```
julia> IntervalBox(X::IntervalBox, n) = n == 1 ? X : X × IntervalBox(X, n-1)
```

works.

Also define `X^n` to mean the same?

**Comment by dpsanders** [comment]
Probably better to use Base.Iterators.repeated or something.

**Comment by eeshan9815** [comment]
Something like this - `IntervalBox(X::IntervalBox, n) = mapreduce(identity, ×, Iterators.repeated(X, n))`?
Both of them give very similar performance.

**Comment by dpsanders** [comment]
Sure, that is good. Or use foldl?

**Comment by eeshan9815** [comment]
```
julia> @btime mapreduce(identity, ×, Iterators.repeated(X, 20));
  86.641 μs (2068 allocations: 128.83 KiB)

julia> @btime foldl(×, Iterators.repeated(X, 20));
  86.758 μs (2068 allocations: 128.83 KiB)
```
They give exactly the same performance, so I think they are call the same function in the backend, the latter looks neater, I think.

**Comment by dpsanders** [comment]
OK great. Remember to put $X when using BenchmarkTooks.jl

**Comment by eeshan9815** [comment]
Added to #166!

**Comment by eeshan9815** [comment]
Fixed by PR #166

**Comment by dpsanders** [comment]
Thanks!
